ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewManager.java

```diff
@@ -154,16 +165,18 @@ public void setPointerEvents(ReactViewGroup view, @Nullable String pointerEvents
 
   @ReactProp(name = "nativeBackgroundAndroid")
   public void setNativeBackground(ReactViewGroup view, @Nullable ReadableMap bg) {
+    mBg = bg;
```

Comment:
Is there a better way to track the current background style object passed into the React Component? I need to keep track of the last background style to reapply the ripple drawable with new border radius styles.

I wasn't sure if  assigning this to a variable was the best approach, but I couldn't find another way to do it.